lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -372,6 +373,13 @@ def get_encounter_date(record: dict, event_type: EventType) -> Optional[str]:
         elif record.get('testing_date'): # from the 'Testing Determination - Internal' instrument
             encounter_date = record.get('testing_date')
 
+        # We have seen cases when the `attestation_date` is not getting set
+        # by REDCap in the `daily_attesation` instrument. Here, we get the date
+        # based on the instance ID of the `daily_attestation` instrument. It's safe
+        # to do this because the Musher computes the instance from the date.
+        if encounter_date == None and is_complete('daily_attestation', record) and record.repeat_instance:
```

Comment:
Minor nit, but it stood out to me: `is None` is widely preferred to `== None` within Python conventions/best practices. The two comparisons are subtly different, and `is None` is more precise for what is generally desired.

This is not merge-blocking at all, as in this case I believe it has no impact, but wanted to note it.

Reply:
Sorry about that. This is from me spending the day in SQL yesterday and switching back to python. I've made the change to `is`.

Reply:
And I added the DET info to the description at the top of the PR. That's the record I used for testing.